src/app/dev/DevToys.Tools/Helpers/XmlHelper.cs

```diff
@@ -39,7 +39,7 @@ internal static bool IsValid(string? input, ILogger logger)
         }
         catch (Exception ex)
         {
-            logger.LogError(ex, "Invalid data detected 'input'", input);
+            logger.LogError(ex, "Invalid data detected.");
```

Comment:
Let's not log user input. Logging them can result in having the user data leaking. Logs can be retrieved by bots and upload them online.

Reply:
Totally agree on this.